ledger/evalindexer.go

```diff
@@ -43,6 +43,7 @@ type indexerLedgerForEval interface {
 	GetAssetCreator(map[basics.AssetIndex]struct{}) (map[basics.AssetIndex]FoundAddress, error)
 	GetAppCreator(map[basics.AppIndex]struct{}) (map[basics.AppIndex]FoundAddress, error)
 	LatestTotals() (ledgercore.AccountTotals, error)
+	LookupKv(basics.Round, string) (*string, error)
```

Comment:
`indexerLedgerConnector.LookupKv()` gains access to `*ledger.Ledger.LookupKv()` via its field `il: indexerLedgerForEval`.